src/modules/networking/tests/NetworkingTests.cpp

```diff
@@ -75,79 +76,79 @@ namespace OSConfig::Platform::Tests
-    " inet6 fe80::5e42:4bf7:dddd:9b0f/64 scope link noprefixroute";
+        "1: docker0: <BROADCAST,UP,LOWER_UP> mtu 65536 qdisc noqueue state UP group default qlen 1000\n"
+        " link/bridge 0a:25:3g:6v:2f:89 brd 00:00:00:00:00:00\n"
+        " inet 172.32.233.234/8 scope global dynamic noprefixroute docker0 valid_lft forever preferred_lft forever\n"
+        " inet6 ::1/128 scope host valid_lft forever preferred_lft forever\n"
+        "2: eth0: <BROADCAST,UP> mtu 65536 qdisc noqueue state DOWN group default qlen 1000\n"
+        " link/ether 00:15:5d:26:cf:89 brd 00:00:00:00:00:00\n"
+        " inet 172.27.181.213/20 scope noprefixroute valid_lft forever preferred_lft forever\n"
+        " inet 10.1.1.2/16 scope global eth0\n"
+        " inet6 fe80::5e42:4bf7:dddd:9b0f/64 scope link noprefixroute";
 
     std::string g_testCommandOutputDefaultGateways =
-    "default via 172.17.128.1 dev docker0 proto\n"
-    " 172.29.64.0/20 dev eth0 proto kernel scope link src 172.29.78.164\n"
-    " default via 172.13.145.1 dev eth0 proto";
+        "default via 172.17.128.1 dev docker0 proto\n"
+        " 172.29.64.0/20 dev eth0 proto kernel scope link src 172.29.78.164\n"
+        " default via 172.13.145.1 dev eth0 proto";
 
     std::string g_testCommandOutputDnsServers =
-    "Link 2 (docker0)\n"
-        "Current Scopes: DNS\n"
-    "DefaultRoute setting: yes\n"
-        "LLMNR setting: yes\n"
-    "MulticastDNS setting: no\n"
-    "DNSOverTLS setting: no\n"
-        "DNSSEC setting: no\n"
-        "DNSSEC supported: no\n"
-    "Current DNS Server: 8.8.8.8\n"
-            "DNS Servers: 8.8.8.8\n"
-                        "172.29.64.1\n"
-            "DNS Domain: mshome.net\n"
-    "Link 3 (eth0)\n"
-        "Current Scopes: DNS\n"
-    "DefaultRoute setting: yes\n"
-        "LLMNR setting: yes\n"
-    "MulticastDNS setting: no\n"
-    "DNSOverTLS setting: no\n"
-        "DNSSEC setting: no\n"
-        "DNSSEC supported: no\n"
-    "Current DNS Server: 172.29.64.1\n"
-            "DNS Servers: 172.29.64.1\n"
-            "DNS Domain: mshome.net\n";
+        "Link 2 (docker0)\n"
+            "Current Scopes: DNS\n"
+        "DefaultRoute setting: yes\n"
+            "LLMNR setting: yes\n"
+        "MulticastDNS setting: no\n"
+        "DNSOverTLS setting: no\n"
+            "DNSSEC setting: no\n"
+            "DNSSEC supported: no\n"
+        "Current DNS Server: 8.8.8.8\n"
+                "DNS Servers: 8.8.8.8\n"
+                            "172.29.64.1\n"
+                "DNS Domain: mshome.net\n"
+        "Link 3 (eth0)\n"
+            "Current Scopes: DNS\n"
+        "DefaultRoute setting: yes\n"
+            "LLMNR setting: yes\n"
+        "MulticastDNS setting: no\n"
+        "DNSOverTLS setting: no\n"
+            "DNSSEC setting: no\n"
+            "DNSSEC supported: no\n"
+        "Current DNS Server: 172.29.64.1\n"
+                "DNS Servers: 172.29.64.1\n"
+                "DNS Domain: mshome.net\n";
```

Comment:
These sub-strings can be aligned at right. And should be... The left " should be all on one column, 4 spaces from left margin in this case